server/src/main/java/ru/otus/server/ClientHandler.java

```diff
@@ -35,6 +38,9 @@ public ClientHandler(Server server, Socket socket) throws IOException {
                             disconnect();
                             break;
                         }
+                        if (msg.equals("/w " + username)){
```

Comment:
if (msg.startsWith("/w " + username)){
                            server.unicastMessage(this, msg);
                         }